Code/Engine/Core/System/Window.h

```diff
@@ -274,6 +293,11 @@ class EZ_CORE_DLL ezWindow : public ezWindowBase
 
   mutable ezWindowHandle m_WindowHandle = ezWindowHandle();
 
+#if EZ_ENABLED(EZ_SUPPORTS_SFML)
+#elif EZ_ENABLED(EZ_PLATFORM_LINUX)
```

Comment:
Shouldn't there be something between the #if and #elif ? At least a #pragma warning or whatever, that this isn't implemented ?

Reply:
No, there shouldn't be anything here. Because this section contains the platform implementation specific members of ezWindow. And in the SFML case we doesn't need any members.
This will be cleaner once glfw goes in and I remove SFML completely.